plugins/modules/user_role_workflow_manager.py

```diff
@@ -3027,15 +3033,24 @@ def delete_user(self, user_params):
             - The function uses the "user_and_roles" family and the "delete_user_api" function from the Cisco Catalyst Center API.
         """
         self.log("delete user with user_params: {0}".format(str(user_params)), "DEBUG")
```

Comment:
self.log("Attempting to delete user with user_params: {0}".format(str(user_params)), "DEBUG")

Reply:
Updated the code accordingly.